runtime/Makefile

```diff
@@ -18,3 +18,15 @@ docker-build:
 
 docker-push: docker-build
 	docker push bandprotocol/runtime:$(VERSION)
+
+gcf:
+	cp $(CURRENT_DIR)/google-cloud-functions/google_cloud_function.py $(CURRENT_DIR)/google-cloud-functions/google_cloud_function.bak
+	cp $(CURRENT_DIR)/google-cloud-functions/main.py $(CURRENT_DIR)/google-cloud-functions/main.bak
+	mv $(CURRENT_DIR)/google-cloud-functions/google_cloud_function.py $(CURRENT_DIR)/google-cloud-functions/main.py
```

Comment:
Didn't to copy requirements.txt from this directory like lambda?
Did zip can run `requests`or `ccxr` package?